tests/unit/Framework/TestCaseTest.php

```diff
@@ -1083,7 +1086,10 @@ public function testRunBareThrowsExceptionWhenTestHasInvalidName($name): void
         $testCase = new TestWithDifferentNames($name);
 
         $this->expectException(Exception::class);
-        $this->expectExceptionMessage('PHPUnit\Framework\TestCase::$name must be a non-blank string.');
+        $this->expectExceptionMessage(sprintf(
+            '%s::$name must be a non-blank string.',
+            TestCase::class
+        ));
```

Comment:
@sebastianbergmann 

How do you feel about changes like these?

Reply:
Makes sense to me.

Reply:
@sebastianbergmann 

I'll adjust all the other places, too!